src/ledger/accountobjects.ts

```diff
@@ -0,0 +1,21 @@
+import {removeUndefined} from '../common'
+import {RippleAPI} from '../api'
+import {AccountObjectsResponse} from '../common/types/commands/account_objects'
+
+export default async function getAccountObjects(
+  this: RippleAPI, address: string, options: any = {}
```

Comment:
since we're pulling off specific properties in this function code below, define the type of the `options` object.

Ex:
```ts
interface GetAccountObjectsOptions {
  type: string,
  ledgerHash?: string,
  ledgerIndex?: string,
  limit?: number
}
```